src/ImageSharp/Processing/Processors/Quantization/WuFrameQuantizer{TPixel}.cs

```diff
@@ -199,27 +155,18 @@ protected override ReadOnlyMemory<TPixel> GetPalette()
             if (this.palette is null)
             {
                 this.palette = new TPixel[this.colors];
-                Span<long> vwtSpan = this.vwt.GetSpan();
-                Span<long> vmrSpan = this.vmr.GetSpan();
-                Span<long> vmgSpan = this.vmg.GetSpan();
-                Span<long> vmbSpan = this.vmb.GetSpan();
-                Span<long> vmaSpan = this.vma.GetSpan();
+                ReadOnlySpan<Moment> momentsSpan = this.moments.GetSpan();
 
                 for (int k = 0; k < this.colors; k++)
                 {
                     this.Mark(ref this.colorCube[k], (byte)k);
 
-                    float weight = Volume(ref this.colorCube[k], vwtSpan);
+                    Moment moment = Volume(ref this.colorCube[k], momentsSpan);
 
-                    if (MathF.Abs(weight) > Constants.Epsilon)
+                    if (moment.Weight > 0)
```

Comment:
I'm not familiar with the algorithm, but removing `Epsilon` changes the algorithm a bit. Are there any corner cases where this may have an impact?

Reply:
None that I could find or test. I know the algorithm pretty well now and the epsilons were actually added as part of some bad design from me in the past. Weight should be a `long`.

Reply:
Cool! If the struct is uniform (all members Int64), it opens up nice SIMD optimization opportunities.